interpreter/src/lib.rs

```diff
@@ -139,6 +139,9 @@ impl Interpreter {
             (Value::Number(lhs), TokenType::Star, Value::Number(rhs)) => {
                 Ok(Value::Number(lhs * rhs))
             }
+            (Value::Number(lhs), TokenType::Percent, Value::Number(rhs)) => {
+                Ok(Value::Number(lhs % rhs))
+            }
```

Comment:
Move this so that it's placed before the `(Value::Number(lhs), TokenType::Plus, Value::Number(rhs))` arm.